library/objective-c/test/BUILD

```diff
@@ -0,0 +1,13 @@
+load("@rules_cc//cc:defs.bzl", "objc_library")
+
+licenses(["notice"])  # Apache 2
+
+objc_library(
```

Comment:
Guessing the reason this doesn't work is because you need an `ios_unit_test` like we have here: https://github.com/envoyproxy/envoy-mobile/blob/0774f4d179940659c7fbdec220ea239ded7864c6/bazel/swift_test.bzl#L32-L36

Reply:
yep, for sure this won't run tests w/o it